packages/universal-components/src/RangeDatePicker/components/ControlContainer.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { format, isSameDay } from 'date-fns';
+
+import { StyleSheet } from '../../PlatformStyleSheet';
+import ControlContainerItem from './ControlContainerItem';
+
+export const LONG_DAY_MONTH_FORMAT = 'EEE d MMM'; // @TODO add to config
```

Comment:
`universal-components` should not have dependency to `margarita-config`. We should probably create new config just for universal components

Reply:
And cannot this string be specified in component props? Something similar like texts/labels passed in `labels` to make it even more variable.

Reply:
It could be specified in props, but I use this component inside of many other components and it would have to bubble across many layers which does not seem me right.

Reply:
As I see you're already passing `label` string in props, which looks like similar case to me.
But if it will be a problem we can probably just add `constants.js` to component root and put this string there (like for example in case of `TagsInput` component). I don't think we will need to reuse it in any other component right now.

Reply:
You are right! I didn't realise that this component is rendered directly in `RangeDatePicker`. In this case it's totally fine, so I added it to the props.

Reply:
👍 Thanks, it looks fine now.